core/creator/src/main/java/io/quarkus/creator/phase/augment/AugmentOutcome.java

```diff
@@ -16,12 +17,14 @@
     private final List<ArtifactResultBuildItem> packageOutput;
     private final JarBuildItem jar;
     private final NativeImageBuildItem nativeImage;
+    private final DockerImageResultBuildItem dockerImage;
```

Comment:
I think this should be in an extension (i.e. there should be no docker stuff in core). Native will likely be moved out to it own extension eventually as well. 

In terms of this particular class the image can be represented by a ArtifactResultBuildItem with type container-image.

Reply:
code has been cleaned from Docker / Container specific stuff.